build.gradle

```diff
@@ -1,27 +1,30 @@
 plugins {
-	id 'org.springframework.boot' version '2.4.0'
-	id 'io.spring.dependency-management' version '1.0.10.RELEASE'
-	id 'java'
+    id 'org.springframework.boot' version '2.4.0'
```

Comment:
Just as comment. Be careful with re-format non edited lines, it complicates the code reviewing and overwrite the line author.